topics/introduction/tutorials/galaxy-onDockstore/galaxy-onDockstore.html

```diff
@@ -0,0 +1,221 @@
+
+.image-40[
+  ![Structure of Dockstore]({{ site.baseurl }}/assets/images/Dockstore-Schematic.png)
+]
+
+[dockstore.org](https://dockstore.org/)
+
+]
+
+---
+
+# What is Dockstore?
+
+---
+
+## Why use Dockstore?
+
+Dockstore follows the FAIR Guiding Principles (Findable, Accessible, Interoperable and Reusable).
+
+## Findable
+
+- Users are strongly suggested to include robust metadata and human readable instructions. Dockstore parses provided metadata and allows it to be searchable.
+
+- The faceted search feature allows users to find relevant workflows using search terms and filter their search using workflow language, author, organization, labels, etc.
+
+- Users can explore workflows from the Organizations and Collections pages where institutions and organizations can showcase their workflows and tools by grouping them by their research theme. For example, IWCG
+
+---
+
+## Why use Dockstore?
+
+Dockstore follows the FAIR Guiding Principles (Findable, Accessible, Interoperable and Reusable).
+
+## Accessible
+
+- Dockstore never requires a user to log in to search and inspect contents for workflows and tools. Links to source repositories are provided allowing users to study the workflows in detail.
+
+- Once found, analyses can be launched in different environments using the Launch with feature of Dockstore. Workflows can just as easily be downloaded to run on local machines.
+
+- Users can also request a DOI via Zenodo for their workflows. This allows researchers to effectively reference the workflows used in published research.
+
+---
+
+## Why use Dockstore?
+
+Dockstore follows the FAIR Guiding Principles (Findable, Accessible, Interoperable and Reusable).
+
+## Interoperable
+
+- Users are strongly encouraged to provide clear instructions, test data and checker workflows to ensure that their workflows are usable in any environment.
+
+- The GA4GH TRS API enable the simple launching of workflows to a variety of compute platforms.
+
+---
+
+## Why use Dockstore?
+
+Dockstore follows the FAIR Guiding Principles (Findable, Accessible, Interoperable and Reusable).
+
+## Reusable
+
+- One of the ultimate goals of Dockstore is to provide workflows and tools that others can find, reuse and build upon. To ensure this, contributors are encouraged to provide clear documentation and the exact version of the components in their descriptor files.
+
+- The Dockstore GitHub App provides support for registering workflows, version control/tagging and automatically syncing workflows and services.
+
+- Ease of access to the workflow allows users to reproduce analyses using the same workflows and even build upon existing workflows.
+
+---
+
+# How to find workflows on [Dockstore](https://dockstore.org/)?
+
+---
+
+pull-left[
+
+## How to find workflows on [Dockstore](https://dockstore.org/)?
+
+You can find relevant workflows using the Faceted Search option:
+
+Scan the code below or simply click on it to be directed to the detailed documentation.
```

Comment:
Would you want to add a slide about the companion portion of "how to find workflows from within galaxy" with just a screenshot of the search box with a search term input or so?

Reply:
Yes! I can add a short section elaborating how to find workflows, using both the search term input as well the TRS ID. I have also recently made two YouTube videos for using Galaxy in Dockstore. What do you think about adding a link to those as well?

Reply:
yes, that sounds great!